src/core/interfaces.ts

```diff
@@ -9,36 +9,27 @@ module Plottable {
   /**
    * Retrieves scaled datum property.
    */
-  export type _Projector = (datum: any, index: number, dataset: Dataset) => any;
+  export type Projector = (datum: any, index: number, dataset: Dataset) => any;
 
   export type AppliedProjector = (datum: any, index: number) => any;
 
   /**
-   * Defines a way how specific attribute needs be retrieved before rendering.
-   */
-  export type _Projection = {
-    accessor: Accessor<any>;
-    scale?: Scale<any, any>;
-    attribute: string;
-  }
-
-  /**
    * A mapping from attributes ("x", "fill", etc.) to the functions that get
    * that information out of the data.
    */
-  export type AttributeToProjector = { [attrToSet: string]: _Projector; };
+  export type AttributeToProjector = { [attrToSet: string]: Projector; };
 
   export type AttributeToAppliedProjector = { [attrToSet: string]: AppliedProjector; };
 
   export type SpaceRequest = {
     minWidth: number;
     minHeight: number;
-  }
+  };
```

Comment:
There is no TSLINT rule for semicolons after `export`. My `Sublime` complains all the time, so I just fix them as I see them